cmd/podman/create.go

```diff
@@ -241,7 +242,7 @@ func parseSecurityOpt(config *createConfig, securityOpts []string) error {
 
 	for _, opt := range securityOpts {
 		if opt == "no-new-privileges" {
-			config.NoNewPrivileges = true
+			config.NoNewPrivs = true
```

Comment:
Does this need to be added to the createConfig at ~line 542 in parseCreateOpts?

Reply:
I believe it is in there.